business_rules_bulk/operators.py

```diff
@@ -123,10 +123,10 @@ def equal_to_case_insensitive(self, other_string):
         values_not_satisfying_equal_to_case_sensitive : list of instances which DOES NOT satisfy the
         equal_to_case_insensitive condition against other_string
         """
-        values_satisfying_equal_to_case_sensitive = filter(
-            lambda single_instance: single_instance[self.value.variable].lower() == other_string.lower() if
-            single_instance[self.value.variable] else False, self.value.instances)
-
+        values_satisfying_equal_to_case_sensitive = [single_instance for single_instance in self.value.instances
```

Comment:
Same as dict.items() now filter() and map() too returns views instead of list. We have to wrap them with the list()
So filter() in py2 would be like list(filter()) in py3, But instead i have removed use of filter along with lambda and used list comprehensions. Old code was not compatible with python 3, So wrapping with list() was done by 2to3 tools but these list comprehensions changes is done by me manually. For tool reference https://docs.python.org/2/library/2to3.html#2to3fixer-filter. In document it also mentioned that wrapping with list is quick fix but the better fix is often to [use a list comprehension](https://docs.python.org/release/3.0/whatsnew/3.0.html#views-and-iterators-instead-of-lists) (especially when the original code uses lambda)